vlib/v/tests/aliased_array_operations.vv

```diff
@@ -0,0 +1,16 @@
+type Ints = []int
```

Comment:
This should be just `aliased_array_operations.v`, not `.vv` .
`.vv` files are skipped normally, and this is a proper test that should compile and run.